### ISSUE TITLE
报错 Error [ERR_REQUIRE_ESM]: require() of ES Module Instead change the require of

### ISSUE THREAD

**Issue by long36708** [unplugin/unplugin-turbo-console#49]
### Describe the bug

require('unplugin-turbo-console/webpack').default({
				extendedPathFileNames: ['index']
			}), 这样写报错 Error [ERR_REQUIRE_ESM]: require() of ES Module Instead change the require of

定位到是 h3包依赖的 crossws升级导致的不兼容 commonjs，临时解决锁定版本  crossws@^0.2.4

### Reproduction

tt

### System Info

```Shell
System:
    OS: Windows 11 10.0.26100
    CPU: (16) x64 AMD Ryzen 7 PRO 4750U with Radeon Graphics
    Memory: 23.85 GB / 47.33 GB
  Binaries:
    Node: 18.20.0 - D:\dev\nodejs\node.EXE
    Yarn: 1.22.19 - D:\dev\nvm\node_global\yarn.CMD
    npm: 10.5.0 - D:\dev\nodejs\npm.CMD
    pnpm: 8.15.5 - D:\dev\nvm\node_global\pnpm.CMD
  Browsers:
    Edge: Chromium (125.0.2535.67)
    Internet Explorer: 11.0.26100.1
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by panghujiajia** [comment]
相同的问题

**Comment by yuyinws** [comment]
Thanks for your report, this bug had fixed on [`v1.10.6`](https://github.com/unplugin/unplugin-turbo-console/releases/tag/v1.10.6)
